app/controllers/application_controller.rb

```diff
@@ -49,7 +53,7 @@ def clear_claim_session
   end
 
   def admin_session_timed_out?
-    admin_signed_in? && session[:last_seen_at] < ADMIN_TIMEOUT_LENGTH_IN_MINUTES.minutes.ago
+    admin_signed_in? && session[:last_seen_at] < admin_timeout_in_minutes.minutes.ago
```

Comment:
That logic would raise an error if `admin_signed_in?` returned `false` because `false <  admin_timeout_in_minutes.minutes.ago` can't be evaluated – false doesn't have a `<` method.

Reply:
Oh course. I misread it. Wondering if a guard clause with `admin_signed_in?` would be better in that case?

```ruby
def admin_session_timed_out?
  return unless admin_signed_in?
  
  session[:last_seen_at] < admin_timeout_in_minutes.minutes.ago
end
```

Reply:
I think guard clauses are useful for avoiding nested conditionals and making it clear early in longer methods when the preconditions. This is a one line piece of Boolean logic though and I'm not convinced using a guard clause would improve it. It would also mean that rather than returning `false` if an admin user wasn't signed in, we'd be returning `nil`, which feels like a regressive step for a predicate method.

Reply:
Yeah, fair enough 👍